### PR TITLE
Problem: batch tx in benchmark is not necessary

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- batch tx is rarely used in production, try to avoid in benchmark.
- normal async tx sending should be fast enough.

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

## Summary by CodeRabbit

- **New Features**
	- Enhanced transaction handling with improved error feedback for individual transactions.
	- Introduced batch processing of transactions, allowing multiple transactions to be sent in a single operation.

- **Bug Fixes**
	- Simplified function signatures for better clarity and consistency.

- **Refactor**
	- Removed unnecessary complexity related to batch transaction construction.
	- Reorganized code structure for improved readability while maintaining functionality.
	- Adjusted configuration settings for improved transaction processing and indexing capabilities.

### CODE REVIEW

**Comment by mmsqe** [comment]
Do we need to prevent sender nonce could conflicts in funding?

**Comment by yihuang** [comment]
> Do we need to prevent sender nonce could conflicts in funding?

no with the benchmark binary.

**Comment by mmsqe** [comment]
> > Do we need to prevent sender nonce could conflicts in funding?
> 
> no with the benchmark binary.

I rmb the chance of getting uniqueness error is higher when try async fund

**Comment by yihuang** [comment]
> > > Do we need to prevent sender nonce could conflicts in funding?
> 
> > 
> 
> > no with the benchmark binary.
> 
> 
> 
> I rmb the chance of getting uniqueness error is higher when try async fund

that error message is expected for now with block-stm, the tx itself should be successful.

File: testground/benchmark/benchmark/utils.py [hunk 1]
```diff
@@ -110,75 +110,19 @@ def send_transaction(w3, tx, acct, wait=True):
...
-                "-y",
-                "--generate-only",
-                **kwargs,
-            )
-        )
-        for signed in signed_txs
+    txhashes = [
+        w3.eth.send_raw_transaction(signed.rawTransaction) for signed in signed_txs
     ]
+    if wait:
```
Comment (mmsqe) [inline testground/benchmark/benchmark/utils.py:121]: validator need enable enable-indexer

Comment (yihuang) [reply testground/benchmark/benchmark/utils.py:121]: make sense

Comment (yihuang) [reply testground/benchmark/benchmark/utils.py:119]: all nodes use evm indexer now.

**Review by mmsqe — APPROVED** [review]

**Pull request merged**